src/Keboola/DbExtractor/DbRetryProxy.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\DbExtractor;
+
+use Psr\Log\LoggerInterface;
+use Retry\RetryProxy;
+use Retry\Policy\RetryPolicyInterface;
+use Retry\Policy\SimpleRetryPolicy;
+use Retry\BackOff\BackOffPolicyInterface;
+use Retry\BackOff\ExponentialBackOffPolicy;
+
+class DbRetryProxy extends RetryProxy
```

Comment:
I don't think it should be named the same, just from different namespace. That may cause some confusion. You'll need to change the "use" statements anyway, so it's not much help IMHO. While making it "Db" prefixed sends a clear message that this is no ordinary RetryProxy.

Reply:
ok, we'll have to update the extractors, but we're doing that anyways.  cool, agreed.